### PR TITLE
Multiple classes

### LINKED ISSUE #1: Multiple classes
There are two training in the different room. The first trainer teaches how to use git. The second one demonstrates how to bake a cake. Second room can have only 6 people at the same time while the first one is unlimited. 

Some implementation details: 
- `learn` and `teach` methods has to accept a `subject` as an argument
- If there is no space in the room, proper message is displayed when a room tries to accept the new student.  We don’t learn a student if it is not in the room.

### PR DESCRIPTION
**Pull request by OlenaKh**
Implementation Task #1 Multiple classes.

Closes #1

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,5 +8,17 @@
...
+            <version>6.14.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>3.9.1</version>
+        </dependency>
+    </dependencies>
 
 </project>
```
Comment (extsoft) [inline pom.xml]: @OlenaKh Please add an empty line at the end of the file.

File: src/main/java/net/comments/sample/TrainingRoom.java [hunk 2]
```diff
@@ -1,22 +1,46 @@
...
-    //public void accept(Person person) {
+    public TrainingRoom(String number, int size) {
+        this.number = number;
+        this.size = size;
+        persons = new Person[size];
+    }
 
-    //     System.out.println(person.name() + " went into the room" + this.number);
-    // }
+//    public void accept(Person person) {
```
Comment (extsoft) [inline src/main/java/net/comments/sample/TrainingRoom.java]: Please remove commented code.

File: src/main/java/net/comments/sample/TrainingRoom.java [hunk 3]
```diff
@@ -1,22 +1,46 @@
 package net.comments.sample;
 
 public class TrainingRoom implements Room {
+    private final String number;
+    private final int size;
```
Comment (extsoft) [inline src/main/java/net/comments/sample/TrainingRoom.java]: Please remove this variable as it is never used.

File: src/main/java/net/comments/sample/TrainingRoom.java [hunk 4]
```diff
@@ -1,22 +1,46 @@
...
-    // private final int size;
-
-    //  public TrainingRoom(String number, int size){
-    //      this.number = number;
-    //     this.size = 10;
-    //  }
+    public TrainingRoom(String number) {
+        this.number = number;
+        this.size = 1000;
+        persons = new Person[size];
```
Comment (extsoft) [inline src/main/java/net/comments/sample/TrainingRoom.java]: Please set array's size right in this line.

**Review by extsoft — CHANGES_REQUESTED** [review]
@OlenaKh please update the code according to the comments.

File: src/main/java/net/comments/sample/TrainingRoom.java (hunk 3 above)
Comment (OlenaKh) [reply src/main/java/net/comments/sample/TrainingRoom.java]: @extsoft  Are you about size? I used it when create object "small/big room".
For the small room: 

For the big room: as it should be unlimited, but arrays cannot be unlimited, I've declared the possible size inside of constructor.

Please explain me where I'm wrong?

File: pom.xml (hunk 1 above)
Comment (OlenaKh) [reply pom.xml]: @extsoft new commit to PR added

File: src/main/java/net/comments/sample/TrainingRoom.java (hunk 2 above)
Comment (OlenaKh) [reply src/main/java/net/comments/sample/TrainingRoom.java]: @extsoft new commit to PR added

File: src/main/java/net/comments/sample/TrainingRoom.java (hunk 3 above)
Comment (extsoft) [reply src/main/java/net/comments/sample/TrainingRoom.java]: I'm talking about class attribute `size`, not about constructor argument. You declared `private final int size` it, but it's never used in the class implementations.

Comment (OlenaKh) [reply src/main/java/net/comments/sample/TrainingRoom.java]: New commit is added. @extsoft  Is it what you mean?

Comment (extsoft) [reply src/main/java/net/comments/sample/TrainingRoom.java]: @OlenaKh Yes, it is!

**Review by extsoft — APPROVED** [review]

**Comment by extsoft** [comment]
@OlenaKh great job!

**Pull request merged**